controllers/install/lookout_controller.go

```diff
@@ -182,3 +199,95 @@ func (r *LookoutReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&installv1alpha1.Lookout{}).
 		Complete(r)
 }
+
+func createLookoutMigrationJob(lookout *installv1alpha1.Lookout) *batchv1.Job {
+	runAsUser := int64(1000)
+	runAsGroup := int64(2000)
+	terminationGracePeriodSeconds := int64(lookout.Spec.TerminationGracePeriodSeconds)
+	allowPrivilegeEscalation := false
+	parallelism := int32(1)
+	completions := int32(1)
+	backoffLimit := int32(0)
+
+	job := batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        lookout.Name,
+			Namespace:   lookout.Namespace,
+			Labels:      AllLabels(lookout.Name, lookout.Labels),
+			Annotations: map[string]string{"checksum/config": GenerateChecksumConfig(lookout.Spec.ApplicationConfig.Raw)},
+		},
+		Spec: batchv1.JobSpec{
+			Parallelism:  &parallelism,
+			Completions:  &completions,
+			BackoffLimit: &backoffLimit,
```

Comment:
I think this says how many times to try your job.  I think you maybe want 1 instead of 0?

Reply:
these are the values form the Helm chart